chore: Init (#1)
ReVanced/revanced-web-manager: README.md

Reviewer: README can be made similar to our other readme's. Take example of ReVanced Manager and ReVanced Patches.

<p align="center">
  <!-- add logo later -->
   <br>
   Continuing the legacy of Vanced
</p>

# 💊 ReVanced Web Manager

![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/ReVanced/revanced-web-manager/release.yml)
![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)

> Web app to use ReVanced on Web

Reviewer: ```suggestion
Web application to use ReVanced
```

## ❓ About

> ReVanced Web Manager is an web app that uses [ReVanced Patcher](https://github.com/revanced/revanced-patcher) to patch Android apps.

Reviewer: ```suggestion
ReVanced Web Manager is a web application that uses [ReVanced Patcher](https://github.com/revanced/revanced-patcher) to patch Android apps.
```
